### DISCUSSION TITLE
Variable names suggestion

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#6537] [Language Ideas] [answered]
I propose putting recommendation of variable names into the functions you author, then IDE can intelligently look at the pattern of your function (what variable name was returned), to suggest to the caller what they should name the variable that the function is return.

Programmers doesn't have to change anything, while this change makes everyone's life easier, AI can aid the process of naming things when what's returned doesn't have a name

**Comment by siegfriedpammer** [comment, marked as answer]
I think you should open an issue in the roslyn repository, as that is the place where IDE features are implemented.
